applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -359,28 +359,9 @@ To perform DFU using the `nRF Connect Device Manager`_ mobile app, complete the
 
    .. tab:: SUIT
 
-      1. Generate the SUIT envelope by building your application with the FOTA support over Bluetooth Low Energy.
-         You can find the generated :file:`root.suit` envelope in the :file:`<build_dir>/DFU` directory.
-         Alternatively, you can use the generated :file:`dfu_suit.zip` package in the :file:`<build_dir>/zephyr` directory.
-      #. Download the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package to your device.
-
-         .. note::
-            `nRF Connect for Desktop`_ does not currently support the FOTA process.
-
-      #. Use the `nRF Connect Device Manager`_ mobile app to update your device with the new firmware.
-
-         a. Ensure that you can access the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package from your phone or tablet.
-         #. In the mobile app, scan and select the device to update.
-         #. Switch to the :guilabel:`Image` tab.
-         #. In the **Firmware Upgrade** section, tap the :guilabel:`SELECT FILE` button and select the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package.
-         #. Tap the :guilabel:`START` button.
-         #. Wait for the DFU to finish and then verify that the application works properly.
-
-      .. note::
-         Support for SUIT updates is available starting from the following versions of the `nRF Connect Device Manager`_ mobile app:
-
-         * Version ``2.0`` on Android.
-         * Version ``1.7`` on iOS.
+      .. include:: /app_dev/device_guides/nrf54h/ug_nrf54h20_suit_fota_update.rst
```

Comment:
@kapi-no Hey Kamil, so if I understood correctly, you are planning to reuse the steps currently in suit_fota_update as is, right? So please copy the content of the lines - 26 to 47 in a file called, for example, suit_fota_update.txt in the /nrf/doc/nrf/includes/ folder. After that, please delete the orphaned rst file. You can then use the include directive to reuse the text from the include file as follows

```
.. include:: /includes/suit_fota_update.txt
```

Reply:
Aligned it according to your suggestion